en/advanced_config/tuning_the_ecl_ekf.md

```diff
@@ -277,7 +296,65 @@ The `hpos_drift_rate`, `vpos_drift_rate` and `hspd` are calculated over a period
 
 Range finder distance to ground is used by a single state filter to estimate the vertical position of the terrain relative to the height datum.
 
-If operating over a flat surface that can be used as a zero height datum, the range finder data can also be used directly by the EKF to estimate height by setting the [EKF2_HGT_MODE](../advanced_config/parameter_reference.md#EKF2_HGT_MODE) parameter to 2.
+The fusion modes of operation are controlled by [EKF2_RNG_CTRL](../advanced_config/parameter_reference.md#EKF2_RNG_CTRL).
+1. [Conditional range aiding](#conditional_range_aid)
+1. [Range height fusion](#range_fusion)
+
+For more details about the configuration of height sources, [click here](#height).
+
+<span id="conditional_range_aid"></span>
+#### Conditional range aiding
+
+Conditional range finder fusion (a.k.a. *Conditional range aid*) activates the range finder fusion for height estimation during low speed/low altitude operation (in addition to the other active height sources).
+If the range finder is set as the reference height source (using [EKF2_HGT_REF](../advanced_config/parameter_reference.md#EKF2_HGT_REF)), the other active height sources such as baro and GNSS altitude will adjust their measurement over time to match the readings of the range finder and when the conditions are not met to start range aiding, a secondary reference is automatically selected.
+
+:::note
+Switching between height references causes the absolute altitude estimate to drift over time. If this is unwanted, it is recommended to set the GNSS altitude as the height reference, even when using conditional range aid.
```

Comment:
It sounds like a problem - why would it be unwanted/when would you decide this was needed?

```suggestion
Switching between height references causes the absolute altitude estimate to drift over time.
If this is unwanted, it is recommended to set the GNSS altitude as the height reference, even when using conditional range aid.
```